src/test/java/org/folio/aes/service/AesServiceTest.java

```diff
@@ -114,32 +125,38 @@ public void testSendTenantNone() throws InterruptedException {
   }
 
   @Test
-  public void testSendRuleException() throws InterruptedException {
+  public void testSendRuleException(@Mock RoutingContext ctx, @Mock HttpServerRequest request,
+      @Mock HttpServerResponse response) throws InterruptedException {
     MultiMap headers = MultiMap.caseInsensitiveMultiMap();
     headers.add(OKAPI_TENANT, "abc");
     headers.add("Content-Type", "application/json");
+    when(ctx.request()).thenReturn(request);
+    when(ctx.response()).thenReturn(response);
     when(request.headers()).thenReturn(headers);
     when(request.params()).thenReturn(MultiMap.caseInsensitiveMultiMap());
     CompletableFuture<Collection<RoutingRule>> cf = new CompletableFuture<>();
     cf.completeExceptionally(new RuntimeException("abc"));
-    when(ruleService.getRules(any(), any(), any())).thenReturn(cf);
+    when(ruleService.getRules(Mockito.any(), any(), any())).thenReturn(cf);
```

Comment:
Curious about Mockito.any(). It seems any() would work too and can remove line 33.

Reply:
Whoops, cut and paste error, missed removing some I guess. 😄